data/categories.txt

```diff
@@ -25,12 +25,16 @@ category "ship"
 	"Superheavy"
 	"Fighter"
 	"Drone"
+	"Unclassified"
+	"Unclassified Minor"
+	"Game Mechanics"
```

Comment:
```suggestion
```

Reply:
I still think there's reason to separate any ships specifically made for a game mechanic to operate, as opposed to ships that exist within universe and are able to be classified or not. These are two very different things - one is code/mechanic based, one is purpose-based within the game.

Reply:
Updated